tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -373,7 +373,7 @@ public:
    {
       return fColumnRanges.find(physicalId) != fColumnRanges.end();
    }
-   std::uint64_t GetBytesOnStorage() const;
+   std::uint64_t GetNBytesOnDisk() const;
```

Comment:
see also comment on previous commit; the binary format specification calls it "storage"...